### PR TITLE
MagickWriter in generic writer

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
To follow the @dcoeurjo PR with the new MagickWriter. We can now export color image using generic writer. 

# Checklist
- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Travis & appveyor)

### CODE REVIEW

**Comment by dcoeurjo** [comment]
🎉🎉

**Comment by kerautret** [comment]
It should also fix travis from the gcc internal error 🌟✨

File: tests/io/writers/testGenericWriter.cpp [hunk 1]
```diff
@@ -117,10 +128,15 @@ bool testGenericWriter()
...
+#ifdef WITH_MAGICK
+  bool ok9 = DGtal::GenericWriter<Image2DColor>::exportFile( "testGenericWriterColorImage.png", anColorImage2D);
+#else
+  bool ok9 = true;
+#endif  
 
-
-
-  nbok += ok1 && okh5 && okh5bis && ok2 && ok3 && ok3bis && ok4 & ok5 & ok6 & ok7 ? 1 : 0; 
+  nbok += ((ok1 && okh5 && okh5bis && ok2 && ok3 && ok3bis && ok4 && ok5 && ok6 && ok7 && ok8 && ok9) ? 1 : 0);
```
Comment (dcoeurjo) [inline tests/io/writers/testGenericWriter.cpp]: at some point, I'd be nice to switch to Catch ;)

**Comment by dcoeurjo** [comment]
THanks. Merging

**Pull request merged**